### PR TITLE
New Script: TeddyCloud Server

### PR DESCRIPTION
**Pull request by dsiebel**
## ✍️ Description

New Scripts to install and update the [TeddyCloud Server](https://tonies-wiki.revvox.de/docs/tools/teddycloud/).

- Related Discussion: https://github.com/community-scripts/ProxmoxVE/discussions/89

---

## 🛠️ Type of Change
Please check the relevant options:  
- [ ] Bug fix (non-breaking change that resolves an issue)  
- [ ] New feature (non-breaking change that adds functionality)  
- [ ] Breaking change (fix or feature that would cause existing functionality to change unexpectedly)  
- [x] New script (a fully functional and thoroughly tested script or set of scripts)  

---

## ✅ Prerequisites
The following steps must be completed for the pull request to be considered:  
- [x] Self-review performed (I have reviewed my code to ensure it follows established patterns and conventions.)  
- [x] Testing performed (I have thoroughly tested my changes and verified expected functionality.)  
- [x] Documentation updated (I have updated any relevant documentation)

---

## 📋 Additional Information (optional)

I just started looking into TeddyCloud and the ecosystem, so I have currently no idea if the setup is sufficient to work with it. Maybe those involved in the discussion can give this a try. FYI @StefanX13 @Goldi95

### CODE REVIEW

File: install/teddycloud-install.sh [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  mc \
+  libubsan1 \
+  ffmpeg \
+  curl \
+  ca-certificates
+msg_ok "Installed Dependencies"
+
+msg_info "Installing TeddyCloud"
+RELEASE="$(curl -s https://api.github.com/repos/toniebox-reverse-engineering/teddycloud/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')"
+VERSION="${RELEASE#tc_v}"
```
Comment (MickLesk) [inline install/teddycloud-install.sh:29]: you can do  for example RELEASE:1 to crop the first. Can you check it? Then you dont need VERSION here

File: json/teddycloud.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    ],
+    "default_credentials": {
+        "username": null,
+        "password": null
+    },
+    "notes": [
+      {
+          "text": "Execute within the Proxmox shell",
+          "type": "info"
+      },
```
Comment (michelroegl-brunner) [inline json/teddycloud.json:37]:
```suggestion
```

Not needed.

File: install/teddycloud-install.sh (hunk 1 above)
Comment (dsiebel) [reply install/teddycloud-install.sh:29]: Yes, I could, but I'd have to do that in multiple places and keep it consistent.
This way, `VERSION` always contains the "clean" tag, while `RELEASE` contains the whole tag.
Both need to be referenced in the download URL for example.

Happy to change it, though, if you insist.

File: json/teddycloud.json (hunk 2 above)
Comment (dsiebel) [reply json/teddycloud.json]: fixed

**Review by MickLesk — APPROVED** [review]

**Review by michelroegl-brunner — APPROVED** [review]

**Pull request merged**